lib/init.coffee

```diff
@@ -1,6 +1,7 @@
-{BufferedProcess, CompositeDisposable} = require 'atom'
+{Directory, BufferedProcess, CompositeDisposable} = require 'atom'
```

Comment:
`BufferedProcess` isn't necessary since the helpers module is doing the actual execution.